tasks/main.yml

```diff
@@ -1,5 +1,13 @@
 ---
 
+- name: Assert Required Variables
+  assert:
+    that:
+      - sansible_php_fpm_description is defined
+      - sansible_php_fpm_user is defined
+      - sansible_php_fpm_group is defined
+      - sansible_php_fpm_chroot is defined
```

Comment:
These variables can be undefined if someone wants to install PHP without FPM, perhaps these should be move to the when clause for the fpm.yml include in build.yml.